src/components/image.tsx

```diff
@@ -17,17 +17,20 @@ export interface ImageProps {
     className?: string;
     /** image alt */
     alt?: string;
+    /** image title */
+    title?: string;
 }
 
 export const Image: React.StatelessComponent<ImageProps> = ({
-    shape, src, size, id, className, alt,
+    shape, src, size, id, className, alt, title,
 }) => {
     const classes = classNames(`image image--${shape}`, className);
     return (
         <img
             id={id}
             src={src}
             alt={alt}
+            title={title}
```

Comment:
I have checked the [prerequisite](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L48) but I do not understand why it's used there.

Reply:
We have an existing `<Image />` component which can be reused in [here](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L48), to replace the `<img />` tag. However in order to replace it while retain its original functionality, the `<Image />` component is missing the attribute `title`, which exist in marketplace_portal (see [here](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L52)).